common/workunit/workunit.cpp

```diff
@@ -1121,8 +1029,66 @@ class CDaliWorkUnit : public CPersistedWorkUnit
             throw;
         }
     }
+    virtual void setGraphState(const char *graphName, WUGraphState state) const
+    {
+        Owned<IRemoteConnection> conn = getWritableProgressConnection(graphName);
+        conn->queryRoot()->setPropInt("@_state", state);
+    }
+    virtual void setNodeState(const char *graphName, WUGraphIDType nodeId, WUGraphState state) const
```

Comment:
Both WUGraphIDType and WUNodeIDType could now become unsigned.  Is there any advantage if they did?